api/db/changelog/db.changelog-115-addColumn-user-create-modified.xml

```diff
@@ -7,17 +7,14 @@
   <changeSet author="nsaxena" id="db.changelog-115-addColumn-user-create-modified">
     <addColumn tableName="user">
       <column name="creation_time" type="datetime">
-        <constraints nullable="false" />
+        <constraints nullable="true" />
       </column>
-      <column name="last_modified_time" type="datetime">
-        <constraints nullable="false"/>
+      <column name="last_modified_time" type="datetime" defaultValueDate="CURRENT_TIMESTAMP">
```

Comment:
do we need a default value here?

We are already allowing nullable and we're copying over the first_sign_in_time (where it exists).

The integrity of `last_modified_time` is not as important but I still think its better to avoid setting it to a mix of values (first_sign_in_time and current timestamp).

If we do want to keep the default value, we can get rid of the nullable constraint.

Reply:
Lsat_modified_time should have some value so in case of null first_sign it would be current date.

Reply:
Liquibase migrations should work the same way every time. If we change the default, we're really changing the schema *every time we run the migration*.